TileFormats/Instanced3DModel/README.md

```diff
@@ -22,61 +22,131 @@ A tile is composed of a header section immediately followed by a body section.
+the instance will default to the `east/north/up` reference frame's orientation for the instance's Cartographic position (`x`, `y`, `z` converted to `longitude` and `latitude` on the `WGS84` ellipsoid).
+
+The `x` vector in the standard basis maps onto the `right` vector in the transformed basis, and the `y` vector maps on to the `up` vector.
+The `z` vector would map onto a `forward` vector, but it is omitted because it will always be the cross product of `right` and `up`.
+
+**Figure 2**: A box in the standard basis.
+
+![box standard basis](figures/box-standard-basis.png)
+
+**Figure 3**: A box transformed into a rotated basis.
+
+![box rotated basis](figures/box-rotated-basis.png)
+
+#### Oct-encoded Normal Vectors
+
+If `NORMAL_UP` and `NORMAL_RIGHT` are not defined for an instance, its orientation may be stored as oct-encoded normals in `NORMAL_UP_OCT32P` and `NORMAL_RIGHT_OCT32P`.
+
+These define `up` and `right` using the oct-encoding described in
+[*A Survey of Efficient Representations of Independent Unit Vectors* by Cigolle et al.](http://jcgt.org/published/0003/02/01/).
+
+An implementation for encoding and decoding these unit vectors can be found in Cesium's
+[AttributeCompression](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Core/AttributeCompression.js)
+module.
+
+### Instance Position
+
+`POSITION` defines the location where an instance should be placed in Cartesian space.
+
+#### Quantized Positions
+
+If `POSITION` is not defined for an instance, its position may be stored in `POSITION_QUANTIZED` which defines the instance position relative to the quantized volume.
+If neither `POSITION` or `POSITION_QUANTIZED` are defined, the instance will not be created.
+
+A quantized volume is defined by `offset` and `scale` to map quantized positions into model space.
+
+**Figure 4**: A quantized volume based on `offset` and `scale`.
+
+![quantized volume](figures/quantized-volume.png)
+
+`offset` is stored in the global semantic `QUANTIZED_VOLUME_OFFSET`, and `scale` is stored in the global semantic `QUANTIZED_VOLUME_SCALE`.
+If those global semantics are not defined, `POSITION_QUANTIZED` cannot be used.
+
+Quantized positions can be mapped to model space using the formula:
+
+`POSITION = POSITION_QUANTIZED * QUANTIZED_VOLUME_SCALE + QUANTIZED_VOLUME_OFFSET`
+
+### Instance Scaling
+
+Scaling can be applied to instances using the `SCALE` and `SCALE_NON_UNIFORM` semantics.
+`SCALE` applies a uniform scale along all axes, and `SCALE_NON_UNIFORM` applies scaling to the `x`, `y`, and `z` axes independently.
+
+If `SCALE` and `SCALE_NON_UNIFORM` are defined for an instance, both scaling operations will be applied.
+
+## Batch Table
+
+Contains metadata organized by `batchId` that can be used for declarative styling.
+
+See the [Batch Table](TODO:add link) reference for more information.
```

Comment:
Should this link just be https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/master/TileFormats/Batched3DModel#batch-table for now?